docs/plans/2026-02-17-faceted-search-counts-design.md

```
@@ -0,0 +1,224 @@
+      "last_year": 56,
+      "older": 2
+    }
+  }
+}
+```
+
+**Note**: `facets` key only present when `include_facets=True`.
+
+## Data Flow
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)